src/Transforms/BottomUp.dfy

```diff
@@ -47,7 +47,7 @@ module Bootstrap.Transforms.BottomUp {
     // - `f(e)` deeply satisfies the post of `f`
   {
     forall e: Expr | Deep.AllChildren_Expr(e, post) && f.requires(e) ::
-      Deep.All_Expr(f(e), post)
+                                                       Deep.All_Expr(f(e), post)
```

Comment:
This looks wrong: `Deep.AllExpr` is the body of the forall, it shouldn't be aligned with `&&`, right?

Reply:
Indeed. This is a bug.